test/test-data.js

```diff
@@ -0,0 +1,191 @@
+  firestore
+    .collection('dev-AcceptedCategories')
+    .doc(ACCCTG_FURNITURE_BANK_CTG_HOUSEHOLD_ITEMS_ID)
+    .set(ACCCTG_FURNITURE_BANK_CTG_HOUSEHOLD_ITEMS);
+  firestore
+    .collection('dev-AcceptedCategories')
+    .doc(ACCCTG_FURNITURE_BANK_CTG_FOOD_ID)
+    .set(ACCCTG_FURNITURE_BANK_CTG_FOOD);
+}
+
+before(function () {
+  // Initialize mock Firestore database
+  console.log('Initializing Mock Firestore...');
+  console.log('Populating Mock Firestore...');
+  populateMockDatabase(firestoreMock);
+  data.setDatabase(firestoreMock);
+  console.log('Mock Database is Set!');
+});
+
+it('GET Request /data/categories : Get all categories', function (done) {
+  request('http://localhost:3000/data/categories', function (error, response, body) {
+    expect(response.statusCode).to.equal(200);
+    const expectedBody = [CTG_FURNITURE_ID, CTG_FOOD_ID, CTG_CLOTHES_ID, CTG_HOUSEHOLD_ITEMS_ID];
+    expect(JSON.parse(body)).to.deep.equal(expectedBody);
+    done();
+  });
+});
+
+it('GET Request /data/acceptedcategories/:id : get an AcceptedCategory', function (done) {
+  request(
+    `http://localhost:3000/data/acceptedcategories/${ACCCTG_FURNITURE_BANK_CTG_FURNITURE_ID}`,
+    function (error, response, body) {
+      expect(response.statusCode).to.equal(200);
+      expect(JSON.parse(response.body)).to.deep.equal(ACCCTG_FURNITURE_BANK_CTG_FURNITURE);
+      done();
+    }
+  );
+});
+
+it('POST Request /data/acceptedcategories/:id : update qualityGuidelines field', function (done) {
+  request.post(
+    {
+      headers: {'content-type': 'application/json'},
+      url: `http://localhost:3000/data/acceptedcategories/${ACCCTG_FURNITURE_BANK_CTG_HOUSEHOLD_ITEMS_ID}`,
+      body: JSON.stringify({qualityGuidelines: ['gently used', 'no damage']}),
+    },
+    function (error, response, body) {
+      expect(response.statusCode).to.equal(201);
+      expect(
+        firestoreMock
+          .collection('dev-AcceptedCategories')
+          .doc(`${ACCCTG_FURNITURE_BANK_CTG_HOUSEHOLD_ITEMS_ID}`)
+          .get()
+          .data().qualityGuidelines
+      ).to.deep.equal(['gently used', 'no damage']);
+      done();
+    }
+  );
+});
+
+it('DELETE Request /data/acceptedcategories/:id : delete an AcceptedCategory', function (done) {
+  request.delete(
+    `http://localhost:3000/data/acceptedcategories/${ACCCTG_FURNITURE_BANK_CTG_FOOD_ID}`,
+    function (error, response, body) {
+      expect(response.statusCode).to.equal(200);
+    }
+  );
+
+  // Check that AcceptedCategory has been deleted
+  request(
+    `http://localhost:3000/data/acceptedcategories/${ACCCTG_FURNITURE_BANK_CTG_FOOD_ID}`,
+    function (error, response, body) {
+      expect(response.statusCode).to.equal(404);
+      done();
+    }
+  );
+});
+
+it('POST Request /data/acceptedcategories/organization/:id : add new AcceptedCategory', function (done) {
+  const oldCollectionLength = Object.keys(firestoreMock._db._collections['dev-AcceptedCategories'])
+    .length;
```

Comment:
Nit:

It's cleaner if you break like

```
const oldCollectionLength =
    Object.keys(firestoreMock._db._collections['dev-AcceptedCategories']).length;
```

Reply:
This line was automatically formatted. If I change the formatting to your suggestion will will fail the **Code Quality Check / quality-check (pull_request)** check.